ogr/services/github.py

```diff
@@ -693,13 +729,31 @@ def _normalize_label_color(color):
             return color[1:]
         return color
 
-    def get_release(self, identifier: int) -> Release:
+    def release_id_from_name(self, name):
```

Comment:
You can use `_release_id_from_name` instead to say that it is not a part of our API and we are using that method only internally.